### DISCUSSION TITLE
Is the Water Balance Model of interest for Hytest?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [hytest-org/hytest#260] [Dataset Questions]
I helped @mewieczo process some large ASCII output files from the Water Balance Model into NetCDF awhile back: 
https://discourse.pangeo.io/t/fixed-width-ascii-500-000-columns-to-zarr/1635/6

Not sure if HyTest is interested in the balance model, but just putting this as a placeholder if it is.

**Comment by rviger** [comment]
@rsignell-usgs let me check on that, Rich.

**Comment by rsignell-usgs** [comment]
ping @rviger-usgs

**Comment by rviger** [comment]
@rsignell-usgs "yes" is the answer to the question. It's being used as data against which to evaluate other simulations but only informally so far, plus the users currently have their own copy. There's no rush for us but we should get to this when we have time. There's an existing copy of the dataset at https://cida.usgs.gov/thredds/catalog/demo/thredds/mows/publicmwb/catalog.html and it will be moved to the NHGF's S3 bucket at some point. This is a link to the USGS Data Release https://www.sciencebase.gov/catalog/item/55fc3f98e4b05d6c4e5029a1.  

Sounds like there is also a plan to update the CONUS application of MWBM but that's still being worked out by the sponsoring project (MAPPNAT). Jacob will let us know on that end, but this is further in the future (i.e., at least FY24).
